packages/cms/pages/resources/create/create.tsx

```diff
@@ -252,7 +273,11 @@ export const CreateResource: React.FunctionComponent = () => {
                     <EuiAccordion
                       id={`__rightArrowAccordionId_${field.name}`}
                       arrowDisplay="right"
-                      buttonContent={field.name}
+                      buttonContent={`${field.name}${
+                        field.creationRules.includes('required') == true
```

Comment:
I recommend always comparing with `===` and not `==`. Also, you can just check `field.creationRules.includes('required')`

Reply:
Okay